src/renderer/containers/Onboard/CreatePassword/CreatePasswordContainer.tsx

```diff
@@ -52,28 +49,26 @@ export class CreatePassword extends Component<Pick<RouteComponentProps, "history
             {
                 inputId: "password",
                 focused: true,
-                onChange: !this.props.path ? this.handleChange : this.handleConfirmPassword,
+                onChange: this.handleChange,
                 placeholder: "Enter password",
                 valid: this.isValid(this.state.errorMessages.password),
                 errorMessage: this.state.errorMessages.password,
             },
-        ];
-        if (!this.props.path) {
-            inputs.push({
+            {
                 inputId: "confirm",
                 onChange: this.handleChange,
                 placeholder: "Confirm password",
                 valid:
                     this.isValid(this.state.errorMessages.confirm) && this.isValid(this.state.errorMessages.password),
                 errorMessage: this.state.errorMessages.confirm,
-            });
-        }
+            },
+        ];
 
         const {errorMessages, loading} = this.state;
         return (
             <>
-                <h1>{!this.props.path ? "Create" : "Validate"} a password</h1>
-                {!this.props.path && <p>You will use this password to unlock applications and keys.</p>}
+                <h1>{!this.props.path ? "Create" : "Change"} a password</h1>
```

Comment:
maybe it's better to pass a real prop here? It's confusing condition and used multiple times through file

Also, "Create a password" and "Change the password" :D